python/sqlflow_submitter/tensorflow/predict.py

```diff
@@ -150,6 +160,42 @@ def eval_input_fn(batch_size, cache=False):
         del pred_dataset
 
     else:
+        if is_pai:
+            model_params["model_dir"] = FLAGS.checkpointDir
+        else:
+            model_params['model_dir'] = save
+        classifier = estimator(**model_params)
+
+        # FIXME(typhoonzero): copied from train.py
+        def pai_maxcompute_input_fn():
```

Comment:
How about move the PAI code to `pai.py`? The current `tensorflow/train.py` includes `Kerase`, `Estimator` and `PAI` code, it's too hard to read ...

Reply:
Will try to polish later